### PR TITLE
Update income-calculator.js

### PR DESCRIPTION
**Pull request by 4bic**
- Adjusted expenditure data changes to reflect 2015/16 Projected Budget

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-taxclock.codeforkenya.org
+
```
Comment (DavidLemayian) [inline CNAME]: @4bic Why did you delete the CNAME?

Comment (4bic) [inline CNAME]: When running tests, I had temporarily altered to clear `page build warning`.
It's now reverted
